homework/list.js

```diff
@@ -0,0 +1,79 @@
+'use strict';
+
+(function(exports) {
+	var ListNote = function(){
+	var _listNoteContent = [];
+	var _wrapper = NULL;
+	};
+  
+	ListNote.prototype = {
+		this._wrapper = document.querySelector('#note-list-wrapper');
+		start() {
+			this.fetchList(function(data) {
+			this.updateList(data);
+			this.drawList();
+			this.preloadFirstNote();
+		});
+		window.addEventListener('click', function(event) {
+			onNoteOpen(event);
+		}).blind(this));
+	}
+
+	
+	onNoteOpen(event) {
+		if (event.target.classList.contains('note-title')) {
```

Comment:
If a function body is wrapped in a conditional statement with only one branch, the `if...else` should be moved into the caller rather than in the callee.